js/src/annotations/osd-region-draw-tool.js

```diff
@@ -115,6 +115,18 @@
       this.svgOverlay.restoreEditedShapes();
     },
 
+    isImageAnnotation: function(annotation) {
```

Comment:
This function will check if the annotation that was passes is an image annotation - one to be displayed on the canvas - or a metadata annotation. The logic is that if we have an annotation with motivation `oa:describing` we don't want to add it to the list of annotations that will be displayed.